src/FileLoader/ImageLoaders/BMPLoader.cs

```diff
@@ -1,13 +1,60 @@
 ﻿using FileLoader.Interfaces;
+using System.IO;
 using System;
+using Shared.Structs;
+using FileLoader.structs;
 
 namespace FileLoader.ImageLoaders
 {
     public class BMPLoader : IImageLoader
     {
         public IImage Load(string path)
+        {            
+            try
+            {
+                var bmpBytes = File.ReadAllBytes(path);
+                var bmpFile = new BMPFile();
+
+                var resolutions = GetBMPResolutions(bmpBytes);
```

Comment:
If you use a tuple, it will be nice to use deconstructor here -> `var (width, height) = GetBMPResolutions(...)`